FIx code generation logic.

geeknoid opened the pull request.

Commit:
Fix the gen-check target.

istio-testing commented:
@geeknoid: The following test **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
gencheck_operator | e655b3a4694a175ec3cc4e3e878918ddd68b6823 | [link](https://prow.istio.io/view/gcs/istio-prow/pr-logs/pull/istio_operator/316/gencheck_operator/1) | `/test gencheck_operator`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>
<!-- test report -->

geeknoid commented:
Needs https://github.com/istio/test-infra/pull/1880 to pass the gencheck job.

geeknoid commented:
@ostromart Oh and this new gencheck job is highlighting the fact that the currently checked in generated files are out of date and need to be regenerated.

ostromart commented on Makefile.core.mk:
```diff
@@ -37,7 +37,7 @@ fmt: format-go
 
 update-charts: installer.sha
-	@scripts/run_update_charts.sh `cat installer.sha`
+	@scripts/run_update_charts.sh `cat installeitr.sha`
```
typo

geeknoid replied:
Whoops. Fixed.

ostromart commented:
The generated files look to be up to date - running make vfsgen on master does not update assets.

istio-testing commented:
@geeknoid: The following test **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
gencheck_operator | 2535c517172b5d1a65e74c1b02778f3678061b39 | [link](https://prow.istio.io/view/gcs/istio-prow/pr-logs/pull/istio_operator/316/gencheck_operator/2) | `/test gencheck_operator`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>
<!-- test report -->

geeknoid commented:
When I do "make gen", I end up with:

        modified:   pkg/apis/istio/v1alpha1/v1alpha1.pb.html
        modified:   pkg/apis/istio/v1alpha1/values_types.pb.go
        modified:   pkg/apis/istio/v1alpha2/v1alpha2.pb.html
        modified:   pkg/vfs/assets.gen.go

geeknoid commented:
I added the changed files to the PR.

istio-testing commented:
@geeknoid: The following tests **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
gencheck_operator | 2535c517172b5d1a65e74c1b02778f3678061b39 | [link](https://prow.istio.io/view/gcs/istio-prow/pr-logs/pull/istio_operator/316/gencheck_operator/2) | `/test gencheck_operator`
lint_operator | 9289ddc71c6feefff8d951f3ea4ca814fbf58f01 | [link](https://prow.istio.io/view/gcs/istio-prow/pr-logs/pull/istio_operator/316/lint_operator/77) | `/test lint_operator`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>
<!-- test report -->

istio-testing commented:
@geeknoid: The following tests **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
lint_operator | 9289ddc71c6feefff8d951f3ea4ca814fbf58f01 | [link](https://prow.istio.io/view/gcs/istio-prow/pr-logs/pull/istio_operator/316/lint_operator/77) | `/test lint_operator`
gencheck_operator | 9289ddc71c6feefff8d951f3ea4ca814fbf58f01 | [link](https://prow.istio.io/view/gcs/istio-prow/pr-logs/pull/istio_operator/316/gencheck_operator/3) | `/test gencheck_operator`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>
<!-- test report -->

istio-testing commented:
@geeknoid: The following tests **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
lint_operator | 9289ddc71c6feefff8d951f3ea4ca814fbf58f01 | [link](https://prow.istio.io/view/gcs/istio-prow/pr-logs/pull/istio_operator/316/lint_operator/77) | `/test lint_operator`
gencheck_operator | 9289ddc71c6feefff8d951f3ea4ca814fbf58f01 | [link](https://prow.istio.io/view/gcs/istio-prow/pr-logs/pull/istio_operator/316/gencheck_operator/3) | `/test gencheck_operator`
mesh_operator | 9289ddc71c6feefff8d951f3ea4ca814fbf58f01 | [link](https://prow.istio.io/view/gcs/istio-prow/pr-logs/pull/istio_operator/316/mesh_operator/77) | `/test mesh_operator`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>
<!-- test report -->

istio-testing commented:
@geeknoid: The following tests **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
lint_operator | 9289ddc71c6feefff8d951f3ea4ca814fbf58f01 | [link](https://prow.istio.io/view/gcs/istio-prow/pr-logs/pull/istio_operator/316/lint_operator/77) | `/test lint_operator`
gencheck_operator | 9289ddc71c6feefff8d951f3ea4ca814fbf58f01 | [link](https://prow.istio.io/view/gcs/istio-prow/pr-logs/pull/istio_operator/316/gencheck_operator/3) | `/test gencheck_operator`
mesh_operator | 9289ddc71c6feefff8d951f3ea4ca814fbf58f01 | [link](https://prow.istio.io/view/gcs/istio-prow/pr-logs/pull/istio_operator/316/mesh_operator/77) | `/test mesh_operator`
test_operator | 9289ddc71c6feefff8d951f3ea4ca814fbf58f01 | [link](https://prow.istio.io/view/gcs/istio-prow/pr-logs/pull/istio_operator/316/test_operator/80) | `/test test_operator`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>
<!-- test report -->

istio-testing commented:
@geeknoid: The following tests **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
lint_operator | 9289ddc71c6feefff8d951f3ea4ca814fbf58f01 | [link](https://prow.istio.io/view/gcs/istio-prow/pr-logs/pull/istio_operator/316/lint_operator/77) | `/test lint_operator`
gencheck_operator | 9289ddc71c6feefff8d951f3ea4ca814fbf58f01 | [link](https://prow.istio.io/view/gcs/istio-prow/pr-logs/pull/istio_operator/316/gencheck_operator/3) | `/test gencheck_operator`
mesh_operator | 9289ddc71c6feefff8d951f3ea4ca814fbf58f01 | [link](https://prow.istio.io/view/gcs/istio-prow/pr-logs/pull/istio_operator/316/mesh_operator/77) | `/test mesh_operator`
test_operator | 9289ddc71c6feefff8d951f3ea4ca814fbf58f01 | [link](https://prow.istio.io/view/gcs/istio-prow/pr-logs/pull/istio_operator/316/test_operator/80) | `/test test_operator`
mandiff_operator | 9289ddc71c6feefff8d951f3ea4ca814fbf58f01 | [link](https://prow.istio.io/view/gcs/istio-prow/pr-logs/pull/istio_operator/316/mandiff_operator/77) | `/test mandiff_operator`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>
<!-- test report -->

geeknoid commented:
@ostromart Note that with the newly generated files included in the PR, a bunch of jobs are failing now...

istio-testing commented:
@geeknoid: The following tests **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
lint_operator | 9289ddc71c6feefff8d951f3ea4ca814fbf58f01 | [link](https://prow.istio.io/view/gcs/istio-prow/pr-logs/pull/istio_operator/316/lint_operator/77) | `/test lint_operator`
gencheck_operator | 9289ddc71c6feefff8d951f3ea4ca814fbf58f01 | [link](https://prow.istio.io/view/gcs/istio-prow/pr-logs/pull/istio_operator/316/gencheck_operator/3) | `/test gencheck_operator`
mesh_operator | 9289ddc71c6feefff8d951f3ea4ca814fbf58f01 | [link](https://prow.istio.io/view/gcs/istio-prow/pr-logs/pull/istio_operator/316/mesh_operator/77) | `/test mesh_operator`
test_operator | 9289ddc71c6feefff8d951f3ea4ca814fbf58f01 | [link](https://prow.istio.io/view/gcs/istio-prow/pr-logs/pull/istio_operator/316/test_operator/80) | `/test test_operator`
mandiff_operator | 844ce7ebcd554ff95fe70f967443d631b473b991 | [link](https://prow.istio.io/view/gcs/istio-prow/pr-logs/pull/istio_operator/316/mandiff_operator/78) | `/test mandiff_operator`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>
<!-- test report -->

istio-testing commented:
@geeknoid: The following tests **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
lint_operator | 9289ddc71c6feefff8d951f3ea4ca814fbf58f01 | [link](https://prow.istio.io/view/gcs/istio-prow/pr-logs/pull/istio_operator/316/lint_operator/77) | `/test lint_operator`
gencheck_operator | 9289ddc71c6feefff8d951f3ea4ca814fbf58f01 | [link](https://prow.istio.io/view/gcs/istio-prow/pr-logs/pull/istio_operator/316/gencheck_operator/3) | `/test gencheck_operator`
test_operator | 9289ddc71c6feefff8d951f3ea4ca814fbf58f01 | [link](https://prow.istio.io/view/gcs/istio-prow/pr-logs/pull/istio_operator/316/test_operator/80) | `/test test_operator`
mandiff_operator | 844ce7ebcd554ff95fe70f967443d631b473b991 | [link](https://prow.istio.io/view/gcs/istio-prow/pr-logs/pull/istio_operator/316/mandiff_operator/78) | `/test mandiff_operator`
mesh_operator | 844ce7ebcd554ff95fe70f967443d631b473b991 | [link](https://prow.istio.io/view/gcs/istio-prow/pr-logs/pull/istio_operator/316/mesh_operator/78) | `/test mesh_operator`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>
<!-- test report -->

istio-testing commented:
@geeknoid: The following tests **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
lint_operator | 9289ddc71c6feefff8d951f3ea4ca814fbf58f01 | [link](https://prow.istio.io/view/gcs/istio-prow/pr-logs/pull/istio_operator/316/lint_operator/77) | `/test lint_operator`
gencheck_operator | 9289ddc71c6feefff8d951f3ea4ca814fbf58f01 | [link](https://prow.istio.io/view/gcs/istio-prow/pr-logs/pull/istio_operator/316/gencheck_operator/3) | `/test gencheck_operator`
mandiff_operator | 844ce7ebcd554ff95fe70f967443d631b473b991 | [link](https://prow.istio.io/view/gcs/istio-prow/pr-logs/pull/istio_operator/316/mandiff_operator/78) | `/test mandiff_operator`
mesh_operator | 844ce7ebcd554ff95fe70f967443d631b473b991 | [link](https://prow.istio.io/view/gcs/istio-prow/pr-logs/pull/istio_operator/316/mesh_operator/78) | `/test mesh_operator`
test_operator | 844ce7ebcd554ff95fe70f967443d631b473b991 | [link](https://prow.istio.io/view/gcs/istio-prow/pr-logs/pull/istio_operator/316/test_operator/81) | `/test test_operator`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>
<!-- test report -->

istio-testing commented:
@geeknoid: The following tests **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
gencheck_operator | 9289ddc71c6feefff8d951f3ea4ca814fbf58f01 | [link](https://prow.istio.io/view/gcs/istio-prow/pr-logs/pull/istio_operator/316/gencheck_operator/3) | `/test gencheck_operator`
mandiff_operator | 844ce7ebcd554ff95fe70f967443d631b473b991 | [link](https://prow.istio.io/view/gcs/istio-prow/pr-logs/pull/istio_operator/316/mandiff_operator/78) | `/test mandiff_operator`
mesh_operator | 844ce7ebcd554ff95fe70f967443d631b473b991 | [link](https://prow.istio.io/view/gcs/istio-prow/pr-logs/pull/istio_operator/316/mesh_operator/78) | `/test mesh_operator`
test_operator | 844ce7ebcd554ff95fe70f967443d631b473b991 | [link](https://prow.istio.io/view/gcs/istio-prow/pr-logs/pull/istio_operator/316/test_operator/81) | `/test test_operator`
lint_operator | 844ce7ebcd554ff95fe70f967443d631b473b991 | [link](https://prow.istio.io/view/gcs/istio-prow/pr-logs/pull/istio_operator/316/lint_operator/78) | `/test lint_operator`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>
<!-- test report -->

istio-testing commented:
@geeknoid: The following tests **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
mandiff_operator | 844ce7ebcd554ff95fe70f967443d631b473b991 | [link](https://prow.istio.io/view/gcs/istio-prow/pr-logs/pull/istio_operator/316/mandiff_operator/78) | `/test mandiff_operator`
mesh_operator | 844ce7ebcd554ff95fe70f967443d631b473b991 | [link](https://prow.istio.io/view/gcs/istio-prow/pr-logs/pull/istio_operator/316/mesh_operator/78) | `/test mesh_operator`
test_operator | 844ce7ebcd554ff95fe70f967443d631b473b991 | [link](https://prow.istio.io/view/gcs/istio-prow/pr-logs/pull/istio_operator/316/test_operator/81) | `/test test_operator`
lint_operator | 844ce7ebcd554ff95fe70f967443d631b473b991 | [link](https://prow.istio.io/view/gcs/istio-prow/pr-logs/pull/istio_operator/316/lint_operator/78) | `/test lint_operator`
gencheck_operator | 844ce7ebcd554ff95fe70f967443d631b473b991 | [link](https://prow.istio.io/view/gcs/istio-prow/pr-logs/pull/istio_operator/316/gencheck_operator/4) | `/test gencheck_operator`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>
<!-- test report -->

geeknoid commented:
/retest

istio-testing commented:
@geeknoid: The following tests **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
mandiff_operator | 844ce7ebcd554ff95fe70f967443d631b473b991 | [link](https://prow.istio.io/view/gcs/istio-prow/pr-logs/pull/istio_operator/316/mandiff_operator/78) | `/test mandiff_operator`
test_operator | 844ce7ebcd554ff95fe70f967443d631b473b991 | [link](https://prow.istio.io/view/gcs/istio-prow/pr-logs/pull/istio_operator/316/test_operator/81) | `/test test_operator`
lint_operator | 844ce7ebcd554ff95fe70f967443d631b473b991 | [link](https://prow.istio.io/view/gcs/istio-prow/pr-logs/pull/istio_operator/316/lint_operator/78) | `/test lint_operator`
gencheck_operator | 844ce7ebcd554ff95fe70f967443d631b473b991 | [link](https://prow.istio.io/view/gcs/istio-prow/pr-logs/pull/istio_operator/316/gencheck_operator/4) | `/test gencheck_operator`
mesh_operator | 844ce7ebcd554ff95fe70f967443d631b473b991 | [link](https://prow.istio.io/view/gcs/istio-prow/pr-logs/pull/istio_operator/316/mesh_operator/79) | `/test mesh_operator`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>
<!-- test report -->

istio-testing commented:
@geeknoid: The following tests **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
mandiff_operator | 844ce7ebcd554ff95fe70f967443d631b473b991 | [link](https://prow.istio.io/view/gcs/istio-prow/pr-logs/pull/istio_operator/316/mandiff_operator/78) | `/test mandiff_operator`
lint_operator | 844ce7ebcd554ff95fe70f967443d631b473b991 | [link](https://prow.istio.io/view/gcs/istio-prow/pr-logs/pull/istio_operator/316/lint_operator/78) | `/test lint_operator`
gencheck_operator | 844ce7ebcd554ff95fe70f967443d631b473b991 | [link](https://prow.istio.io/view/gcs/istio-prow/pr-logs/pull/istio_operator/316/gencheck_operator/4) | `/test gencheck_operator`
mesh_operator | 844ce7ebcd554ff95fe70f967443d631b473b991 | [link](https://prow.istio.io/view/gcs/istio-prow/pr-logs/pull/istio_operator/316/mesh_operator/79) | `/test mesh_operator`
test_operator | 844ce7ebcd554ff95fe70f967443d631b473b991 | [link](https://prow.istio.io/view/gcs/istio-prow/pr-logs/pull/istio_operator/316/test_operator/82) | `/test test_operator`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>
<!-- test report -->

istio-testing commented:
@geeknoid: The following tests **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
mandiff_operator | 844ce7ebcd554ff95fe70f967443d631b473b991 | [link](https://prow.istio.io/view/gcs/istio-prow/pr-logs/pull/istio_operator/316/mandiff_operator/78) | `/test mandiff_operator`
gencheck_operator | 844ce7ebcd554ff95fe70f967443d631b473b991 | [link](https://prow.istio.io/view/gcs/istio-prow/pr-logs/pull/istio_operator/316/gencheck_operator/4) | `/test gencheck_operator`
mesh_operator | 844ce7ebcd554ff95fe70f967443d631b473b991 | [link](https://prow.istio.io/view/gcs/istio-prow/pr-logs/pull/istio_operator/316/mesh_operator/79) | `/test mesh_operator`
test_operator | 844ce7ebcd554ff95fe70f967443d631b473b991 | [link](https://prow.istio.io/view/gcs/istio-prow/pr-logs/pull/istio_operator/316/test_operator/82) | `/test test_operator`
lint_operator | 844ce7ebcd554ff95fe70f967443d631b473b991 | [link](https://prow.istio.io/view/gcs/istio-prow/pr-logs/pull/istio_operator/316/lint_operator/79) | `/test lint_operator`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>
<!-- test report -->

istio-testing commented:
@geeknoid: The following tests **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
gencheck_operator | 844ce7ebcd554ff95fe70f967443d631b473b991 | [link](https://prow.istio.io/view/gcs/istio-prow/pr-logs/pull/istio_operator/316/gencheck_operator/4) | `/test gencheck_operator`
mesh_operator | 844ce7ebcd554ff95fe70f967443d631b473b991 | [link](https://prow.istio.io/view/gcs/istio-prow/pr-logs/pull/istio_operator/316/mesh_operator/79) | `/test mesh_operator`
test_operator | 844ce7ebcd554ff95fe70f967443d631b473b991 | [link](https://prow.istio.io/view/gcs/istio-prow/pr-logs/pull/istio_operator/316/test_operator/82) | `/test test_operator`
lint_operator | 844ce7ebcd554ff95fe70f967443d631b473b991 | [link](https://prow.istio.io/view/gcs/istio-prow/pr-logs/pull/istio_operator/316/lint_operator/79) | `/test lint_operator`
mandiff_operator | 844ce7ebcd554ff95fe70f967443d631b473b991 | [link](https://prow.istio.io/view/gcs/istio-prow/pr-logs/pull/istio_operator/316/mandiff_operator/79) | `/test mandiff_operator`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>
<!-- test report -->

istio-testing commented:
@geeknoid: The following tests **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
mesh_operator | 844ce7ebcd554ff95fe70f967443d631b473b991 | [link](https://prow.istio.io/view/gcs/istio-prow/pr-logs/pull/istio_operator/316/mesh_operator/79) | `/test mesh_operator`
test_operator | 844ce7ebcd554ff95fe70f967443d631b473b991 | [link](https://prow.istio.io/view/gcs/istio-prow/pr-logs/pull/istio_operator/316/test_operator/82) | `/test test_operator`
lint_operator | 844ce7ebcd554ff95fe70f967443d631b473b991 | [link](https://prow.istio.io/view/gcs/istio-prow/pr-logs/pull/istio_operator/316/lint_operator/79) | `/test lint_operator`
mandiff_operator | 844ce7ebcd554ff95fe70f967443d631b473b991 | [link](https://prow.istio.io/view/gcs/istio-prow/pr-logs/pull/istio_operator/316/mandiff_operator/79) | `/test mandiff_operator`
gencheck_operator | 844ce7ebcd554ff95fe70f967443d631b473b991 | [link](https://prow.istio.io/view/gcs/istio-prow/pr-logs/pull/istio_operator/316/gencheck_operator/5) | `/test gencheck_operator`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>
<!-- test report -->

ostromart commented:
Ah ok, it that's because values_types.proto was modified recently and gen was not run as part of that. Let me take care of it now. I wonder if there's some presubmit check we could add to enforce it?

geeknoid commented:
@ostromart That's specifically what my PR is trying to enforce :-) We will have a gate to prevent a PR from going in without updating generated files.

istio-testing commented:
@geeknoid: The following tests **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
mesh_operator | 844ce7ebcd554ff95fe70f967443d631b473b991 | [link](https://prow.istio.io/view/gcs/istio-prow/pr-logs/pull/istio_operator/316/mesh_operator/79) | `/test mesh_operator`
test_operator | 844ce7ebcd554ff95fe70f967443d631b473b991 | [link](https://prow.istio.io/view/gcs/istio-prow/pr-logs/pull/istio_operator/316/test_operator/82) | `/test test_operator`
lint_operator | 844ce7ebcd554ff95fe70f967443d631b473b991 | [link](https://prow.istio.io/view/gcs/istio-prow/pr-logs/pull/istio_operator/316/lint_operator/79) | `/test lint_operator`
mandiff_operator | 844ce7ebcd554ff95fe70f967443d631b473b991 | [link](https://prow.istio.io/view/gcs/istio-prow/pr-logs/pull/istio_operator/316/mandiff_operator/79) | `/test mandiff_operator`
gencheck_operator | 41787d6e2c896773288cacd266c50b633a06d741 | [link](https://prow.istio.io/view/gcs/istio-prow/pr-logs/pull/istio_operator/316/gencheck_operator/6) | `/test gencheck_operator`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>
<!-- test report -->

istio-testing commented:
@geeknoid: The following tests **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
lint_operator | 844ce7ebcd554ff95fe70f967443d631b473b991 | [link](https://prow.istio.io/view/gcs/istio-prow/pr-logs/pull/istio_operator/316/lint_operator/79) | `/test lint_operator`
gencheck_operator | 7ef80214ddb5afb7a75219090b287b30bff4260d | [link](https://prow.istio.io/view/gcs/istio-prow/pr-logs/pull/istio_operator/316/gencheck_operator/7) | `/test gencheck_operator`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>
<!-- test report -->

istio-testing commented:
@geeknoid: The following test **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
gencheck_operator | a5a6446dae530cf689da1a43a58a3333bcf9c504 | [link](https://prow.istio.io/view/gcs/istio-prow/pr-logs/pull/istio_operator/316/gencheck_operator/8) | `/test gencheck_operator`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>
<!-- test report -->

istio-testing commented:
@geeknoid: The following tests **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
gencheck_operator | a5a6446dae530cf689da1a43a58a3333bcf9c504 | [link](https://prow.istio.io/view/gcs/istio-prow/pr-logs/pull/istio_operator/316/gencheck_operator/8) | `/test gencheck_operator`
lint_operator | a5a6446dae530cf689da1a43a58a3333bcf9c504 | [link](https://prow.istio.io/view/gcs/istio-prow/pr-logs/pull/istio_operator/316/lint_operator/82) | `/test lint_operator`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>
<!-- test report -->

istio-testing commented:
@geeknoid: The following tests **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
gencheck_operator | a5a6446dae530cf689da1a43a58a3333bcf9c504 | [link](https://prow.istio.io/view/gcs/istio-prow/pr-logs/pull/istio_operator/316/gencheck_operator/8) | `/test gencheck_operator`
lint_operator | a5a6446dae530cf689da1a43a58a3333bcf9c504 | [link](https://prow.istio.io/view/gcs/istio-prow/pr-logs/pull/istio_operator/316/lint_operator/82) | `/test lint_operator`
mesh_operator | a5a6446dae530cf689da1a43a58a3333bcf9c504 | [link](https://prow.istio.io/view/gcs/istio-prow/pr-logs/pull/istio_operator/316/mesh_operator/82) | `/test mesh_operator`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>
<!-- test report -->

istio-testing commented:
@geeknoid: The following tests **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
gencheck_operator | a5a6446dae530cf689da1a43a58a3333bcf9c504 | [link](https://prow.istio.io/view/gcs/istio-prow/pr-logs/pull/istio_operator/316/gencheck_operator/8) | `/test gencheck_operator`
lint_operator | a5a6446dae530cf689da1a43a58a3333bcf9c504 | [link](https://prow.istio.io/view/gcs/istio-prow/pr-logs/pull/istio_operator/316/lint_operator/82) | `/test lint_operator`
mesh_operator | a5a6446dae530cf689da1a43a58a3333bcf9c504 | [link](https://prow.istio.io/view/gcs/istio-prow/pr-logs/pull/istio_operator/316/mesh_operator/82) | `/test mesh_operator`
mandiff_operator | a5a6446dae530cf689da1a43a58a3333bcf9c504 | [link](https://prow.istio.io/view/gcs/istio-prow/pr-logs/pull/istio_operator/316/mandiff_operator/82) | `/test mandiff_operator`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>
<!-- test report -->

istio-testing commented:
@geeknoid: The following tests **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
gencheck_operator | a5a6446dae530cf689da1a43a58a3333bcf9c504 | [link](https://prow.istio.io/view/gcs/istio-prow/pr-logs/pull/istio_operator/316/gencheck_operator/8) | `/test gencheck_operator`
lint_operator | a5a6446dae530cf689da1a43a58a3333bcf9c504 | [link](https://prow.istio.io/view/gcs/istio-prow/pr-logs/pull/istio_operator/316/lint_operator/82) | `/test lint_operator`
mesh_operator | a5a6446dae530cf689da1a43a58a3333bcf9c504 | [link](https://prow.istio.io/view/gcs/istio-prow/pr-logs/pull/istio_operator/316/mesh_operator/82) | `/test mesh_operator`
mandiff_operator | a5a6446dae530cf689da1a43a58a3333bcf9c504 | [link](https://prow.istio.io/view/gcs/istio-prow/pr-logs/pull/istio_operator/316/mandiff_operator/82) | `/test mandiff_operator`
test_operator | a5a6446dae530cf689da1a43a58a3333bcf9c504 | [link](https://prow.istio.io/view/gcs/istio-prow/pr-logs/pull/istio_operator/316/test_operator/85) | `/test test_operator`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>
<!-- test report -->

istio-testing commented:
@geeknoid: The following tests **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
lint_operator | a5a6446dae530cf689da1a43a58a3333bcf9c504 | [link](https://prow.istio.io/view/gcs/istio-prow/pr-logs/pull/istio_operator/316/lint_operator/82) | `/test lint_operator`
mesh_operator | a5a6446dae530cf689da1a43a58a3333bcf9c504 | [link](https://prow.istio.io/view/gcs/istio-prow/pr-logs/pull/istio_operator/316/mesh_operator/82) | `/test mesh_operator`
mandiff_operator | a5a6446dae530cf689da1a43a58a3333bcf9c504 | [link](https://prow.istio.io/view/gcs/istio-prow/pr-logs/pull/istio_operator/316/mandiff_operator/82) | `/test mandiff_operator`
test_operator | a5a6446dae530cf689da1a43a58a3333bcf9c504 | [link](https://prow.istio.io/view/gcs/istio-prow/pr-logs/pull/istio_operator/316/test_operator/85) | `/test test_operator`
gencheck_operator | 084ecdf011aa8ca4f3d7b39d12b80ca166ac8f5e | [link](https://prow.istio.io/view/gcs/istio-prow/pr-logs/pull/istio_operator/316/gencheck_operator/9) | `/test gencheck_operator`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>
<!-- test report -->
